contributors/chairs-and-techleads/leadership-changes.md

```diff
@@ -0,0 +1,27 @@
+# Leadership Changes
+
+This document covers steps needed to propose changes to SIG/WG/UG leadership.
+
+- [ ] Discuss the proposed changes with the current leadership.
+
+- [ ] Send an email to the SIG/WG/UG mailing list and cc the [kubernetes-dev]
+mailing list. At a minimum, the email should contain:
+  - [ ] intent to step down as the current lead
+  - [ ] if nominating another lead
+    - [ ] 1-2 lines about why they are being nominated
+    - [ ] contacts (usually current leads) to privately reach out
+          to for questions or concerns about the nomination
+  - [ ] if this was discussed in a meeting, link to meeting notes
+  - [ ] lazy consensus deadline of at least 2 business days
```

Comment:
if we are looking for patterns else where :) Yes, 3 business days or 72 hours sounds good

```PMCs SHOULD ensure that decision-making processes allow input for a sufficient amount of time - typically at least 72 hours - so that project participants in various time zones have a chance to participate in the decision.``` (from https://www.apache.org/dev/pmc.html)

Reply:
Given that most SIGs meet on a weekly or bi-weekly cadence, should we just settle on 1w?

Reply:
+1 to 1w

Reply:
An issue with lazy consensus is it requires active objection to know it was not reached.  If concerns are shared privately with existing leads (or separately to steering-private), we may want to provide additional buffer beyond 72 hours to ensure any communication is handled and concerns are satisfied.  I think a week is probably sufficient.

Reply:
if concerns are raised to the current leads or to steering, it would make sense for the leads or steering to `/hold` the PR until those are resolved.

Reply:
I think 1 week is an acceptable timeframe for leadership changes. It's an infrequent and large enough event to warrant a longer feedback period.